test/utils/net.go

```diff
@@ -104,7 +104,7 @@ func WaitPlatformOnline(t *testing.T) {
 }
 
 // RequirePortOpen checks if the local port(s) accepts connections
-func RequirePortOpen(t *testing.T, ports ...string) {
+func requirePortOpen(t *testing.T, ports ...string) {
```

Comment:
The comment(`RequirePortOpen`) for this internal function(`requirePortOpen`) is not corresponding anymore.